src/index.ts

```diff
@@ -52,26 +52,28 @@ async function runAnalysis() {
   appendFileSync(getRequiredEnvVariable('GITHUB_ENV'), `LACEWORK_TOOLS=${tools.join(',')}\n`)
   const indirectDeps = getInput('eval-indirect-dependencies')
   const toUpload: string[] = []
-  if (tools.includes('sast')) {
+  if (tools.includes('sast') && !tools.includes('sca')) {
```

Comment:
@linpengzhang , `sca` will be run as a single command in the other `if` branch below. This case, where `sast` is enabled but not `sca` is just for backward compatibility. This case is not documented anywhere is I suspect nobody is using it but is only to avoid surprise crashes as we progressively phase out the `tools` option from the GitHub action code.